recipes/bond/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, CMake, tools
+import platform, os
+
+
+class BondConan(ConanFile):
+    name = "bond"
+    version = "9.0.0"
+    license = "MIT License"
+    homepage = "https://github.com/microsoft/bond.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Bond is a cross-platform framework for working with schematized data. It supports cross-language de/serialization and powerful generic mechanisms for efficiently manipulating data. Bond is broadly used at Microsoft in high scale services."
+    topics = ("bond", "microsoft")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [False, True],
+        "fPIC": [True, False]}
+    default_options = {
+        "shared": False,
+        "fPIC": True}
+    build_requires = "boost/1.71.0@conan/stable"
+    generators = "cmake"
+    exports_sources = ["FindBond.cmake"]
```

Comment:
Please remove this file (and export a conan wrapper `CMakeLists.txt` file instead).

As a consumer of this recipe, when using the `cmake_find_package` generator, conan will genere a `FindBond.cmake` script automatically.